packages/cli/src/utils/generate.ts

```diff
@@ -78,6 +79,18 @@ function copyCoreFiles() {
   }
 }
 
+async function copyCypressFiles() {
+  try {
+    const userStoreConfig = await import(userStoreConfigFileDir)
+    if (userStoreConfig.experimental.enableCypressExtension) {
```

Comment:
Did you test to see what happens if there's no `experimental` object on the `userStoreConfig`? I think that it will break. Can you check or correct me?

Reply:
I have the same feeling, i think it would be feasible to have a fallback when there is not custom tests